go/pkg/bertyprotocol/api_attachment.go

```diff
@@ -0,0 +1,327 @@
+package bertyprotocol
```

Comment:
Could you keep in this file only the main API functions and move all the helpers to an `attachment_utils.go` file? (to be consistent with other api methods implem)

Could you also define most of your errors in errcode.proto or at least wrap them in an `errcode.TODO`?